library/std/src/Std/DurrHoyerLibrary.qs

```diff
@@ -0,0 +1,226 @@
+open Microsoft.Quantum.Intrinsic;
+open Microsoft.Quantum.Canon;
+open Microsoft.Quantum.Math;
+open Microsoft.Quantum.Measurement;
+open Microsoft.Quantum.Arrays;
+open Microsoft.Quantum.Convert;
+open Microsoft.Quantum.Random;
+open Microsoft.Quantum.Core;
+open Microsoft.Quantum.Diagnostics;
```

Comment:
```suggestion
import Std.Math;
import Std.Arrays;
import Std.Convert;
import Std.Random;
open Std.Diagnostics;
```
We can remove some of imports here since they are part of prelude(ie  included in Q# by Default). Also, its recommended to import from the Std namespace.

 See [this code](https://github.com/microsoft/qsharp/blob/main/compiler/qsc_data_structures/src/namespaces.rs#L10-L15) for the list of preludes

pub const PRELUDE: &[&[&str]; 4] = &[
    &["Std", "Canon"],
    &["Microsoft", "Quantum", "Core"],
    &["Std", "Intrinsic"],
    &["Std", "Measurement"],
];

Reply:
Ah I glossed over that pattern in the other files. thanks for sharing how those preludes come into play. I was confused by that Bec cause I didn't see a Quantum namespace displayed anywhere on the other Std files.